Retrie/Expr.hs

```diff
@@ -1,4 +1,4 @@
--- Copyright (c) Facebook, Inc. and its affiliates.
+-- CopAAyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Can you please revert this change.

Reply:
oops. reverted it. Thanks for catching that!